natchez-extras-fs2/src/test/scala/com/ovoenergy/natchez/extras/fs2/AllocatedSpanTest.scala

```diff
@@ -62,7 +54,7 @@ class AllocatedSpanTest extends AnyWordSpec with Matchers {
               .covary[IO]
               .through(AllocatedSpan.create(maxOpen = 10)(_ => ep.root("root")))
               .evalTap(_.span.span("foo").use(_ => IO.unit))
-              .metered(1.millisecond)
+              .metered(5.millisecond)
```

Comment:
are you sure?

Reply:
yes I think there's just some flakiness around things finishing absolutely immediately in these test cases hence this